templates/flows/flow_results.haml

```diff
@@ -648,6 +652,34 @@
 
           %tbody{id:"results_run_table"}
             %tr{ic-append-from:"/flow/run_table/{{object.id}}", ic-trigger-on:"load", ic-target:"#results_run_table", ic-indicator:"#indicator"}
+        
+    #links.tab-pane
+      - if trackable_links
+        %table
+            %tr
+              %td.header
+                .details
+                  .text
+                    -trans "Contact"
+              %td.header
+                .details
+                  .text
+                    -trans "Link"
+              %td.header
+
+            %tbody{id:"results_run_table"}
+              - for link in trackable_links
+                %tr
```

Comment:
I think the Intercooler (https://github.com/greatnonprofits-nfp/rapidpro/pull/107/files#diff-af95e1e2a76506fbedc073eb88c51f40R654) should be applied here as well since we can have a lot of results on this loop to be paginated

Reply:
Refactored to use Intercooler.